autoload/context/util.vim

```diff
@@ -1,3 +1,9 @@
+function! context#util#active() abort
+    return 1
+                \ && g:context.enabled
+                \ && !get(g:context.filetype_blacklist, &filetype)
```

Comment:
May I ask why the return statement includes a `1`? It seems a little redundant. It is nothing more than a stylistic choice?

Reply:
Yes, it's just stylistic to have all the real conditions properly aligned 🙃